extensions/amp-selector/1.0/test/test-component.js

```diff
@@ -362,17 +362,17 @@ describes.sandboxed('Selector preact component', {}, () => {
       beforeEach(() => {
         ref = Preact.createRef();
         wrapper = mount(
-          <Selector ref={ref} defaultValue={['a']}>
-            <Option key={1} option="a" index={1}>
+          <BentoSelector ref={ref} defaultValue={['a']}>
+            <BentoSelectorOption key={1} option="a" index={1}>
               option a
-            </Option>
-            <Option key={2} option="b" index={2}>
+            </BentoSelectorOption>
+            <BentoSelectorOption key={2} option="b" index={2}>
               option b
-            </Option>
-            <Option key={3} option="c" disabled index={3}>
+            </BentoSelectorOption>
+            <BentoSelectorOption key={3} option="c" disabled index={3}>
               option c
-            </Option>
-          </Selector>
+            </BentoSelectorOption>
+          </BentoSelector>
         );
 
         const options = wrapper.find(Option);
```

Comment:
These need to be replaced for the unit tests
```suggestion
        const options = wrapper.find(BentoOption);
```